scripts/plugins/storage/postgres/init.sql

```diff
@@ -853,15 +853,23 @@ INSERT INTO fledge.statistics ( key, description, value, previous_value )
 
 -- Core Tasks
 --
-INSERT INTO fledge.scheduled_processes ( name, script ) VALUES ( 'purge',               '["tasks/purge"]'      );
-INSERT INTO fledge.scheduled_processes ( name, script ) VALUES ( 'stats collector',     '["tasks/statistics"]' );
-INSERT INTO fledge.scheduled_processes ( name, script ) VALUES ( 'FledgeUpdater',      '["tasks/update"]'     );
+INSERT INTO fledge.scheduled_processes ( name, script ) VALUES ( 'purge',               '["tasks/purge"]'       );
+INSERT INTO fledge.scheduled_processes ( name, script ) VALUES ( 'stats collector',     '["tasks/statistics"]'  );
+INSERT INTO fledge.scheduled_processes ( name, script ) VALUES ( 'FledgeUpdater',       '["tasks/update"]'      );
 INSERT INTO fledge.scheduled_processes ( name, script ) VALUES ( 'certificate checker', '["tasks/check_certs"]' );
 
 -- Storage Tasks
 --
 INSERT INTO fledge.scheduled_processes (name, script) VALUES ('backup',  '["tasks/backup"]'  );
 INSERT INTO fledge.scheduled_processes (name, script) VALUES ('restore', '["tasks/restore"]' );
+
+-- South, Notification, North Tasks
+--
+INSERT INTO fledge.scheduled_processes (name, script)   VALUES ( 'south_c',        '["services/south_c"]'        );
+INSERT INTO fledge.scheduled_processes (name, script)   VALUES ( 'notification_c', '["services/notification_c"]' );
+INSERT INTO fledge.scheduled_processes (name, script)   VALUES ( 'north_c',        '["tasks/north_c"]'           );
+INSERT INTO fledge.scheduled_processes (name, script)   VALUES ( 'north',          '["tasks/north"]'             );
+
```

Comment:
AFAIK, we never had this before in schema and these entries were created dynamically since beginning when we hit POST service and task API.
And Rest API already have that check for the process name existence, if does not exist then create entry otherwise skip and move further. Also custom Job ran successfully which means System and unit tests are also happy with new changes.